overrides/default.nix

```diff
@@ -432,6 +432,11 @@ lib.composeManyExtensions [
         '';
       });
 
+      cysystemd = super.cysystemd.overridePythonAttrs (old: {
+        buildInputs = (old.buildInputs or [ ]) ++ [ super.setuptools pkgs.systemd ];
```

Comment:
Could you add setuptools through the json file?

Reply:
Okay, sure. I wasn't sure what was preferred for that case.